analyzer/codechecker_analyzer/analyzers/clangsa/statistics_manager.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+"""
+Calculates call statistics from analysis output
```

Comment:
This module just creates the stats collecting clangsa command right now does not process the analysis output.
Maybe the statistics.py filename could be enough too.